pkg/operator/controller/controller.go

```diff
@@ -37,6 +37,10 @@ const (
 	IngressControllerFinalizer = "ingresscontroller.operator.openshift.io/finalizer-ingresscontroller"
 )
 
+const (
+	controllerName = "ingress_controller"
+)
```

Comment:
Might as well join the two `const` blocks.

Reply:
Done